en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md

```diff
@@ -0,0 +1,224 @@
+
+Follow the steps below to encrypt passwords:
+
+1. Add the following `[secrets]` configurations to the bottom of the `deployment.toml` file in the `<IS_HOME>/repository/conf/` directory. Provide an alias for the password type followed by the actual password enclosed within square brackets `[]` as shown below.
+
+    ```toml
+    [secrets]
+    admin_password = "[password_1]"
+    keystore_password = "[password_2]"
+    key_password = "[password_3]"
+    truststrore_password = "[password_4]"
+    log4j2_password = "[password_5]"
+    ```
+
+2.  Open a terminal, navigate to the `<IS_HOME>/bin/` directory, and execute the following command based on the encryption method and your operating system:
+
+    === "Symmetric encryption"
+
+        !!! note "Prerequisite"
+
+            To support symmetric encryption, you should have a symmetric secret in an internal keystore of type PKCS12. Follow the instructions [here]({{base_path}}/deploy/security/keystores/configure-keystores/#add-a-symmetric-secret-to-a-pkcs12-keystore) to add one.
+
+        - **For Linux**: `./ciphertool.sh -Dconfigure -Dsymmetric`
+
+        - **For Windows**: `ciphertool.bat -Dconfigure -Dsymmetric`
+
+    === "Asymmetric encryption"
+
+        !!! warning
+
+            Asymmetric encryption methods like RSA are not recommended due to their vulnerability to post-quantum threats.
+
+        - **For Linux**: `./ciphertool.sh -Dconfigure`
+
+        - **For Windows**: `ciphertool.bat -Dconfigure`
+
+3. Open the `deployment.toml` again and see that the passwords are encrypted.
+
+    ```toml
+    [secrets]
+    admin_password = "xxxxxxx"
+    keystore_password = "xxxxxxx"
+    key_password = "xxxxxxx"
+    truststrore_password = "xxxxxxx"
+    "log4j.appender.LOGEVENT.password" = "xxxxxxx"
+    ```
+
+{% else %}
+
+To encrypt passwords on the WSO2 Identity Server:
+
+1. Add the following `[secrets]` configurations to the bottom of the `deployment.toml` file in the `<IS_HOME>/repository/conf/` directory. Give an alias for the password type followed by the actual password. The following example lists the most common passwords in configuration files.
+
+    ```toml
+    [secrets]
+    admin_password = "[password_1]"
+    keystore_password = "[password_2]"
+    key_password = "[password_3]"
+    truststrore_password = "[password_4]"
+    log4j2_password = "[password_5]"
+    ```
+
+2. Open a terminal, navigate to the `<IS_HOME>/bin/` directory, and execute the following command to enable the cipher tool using the `-Dconfigure` flag.
+
+    - On Linux: `./ciphertool.sh -Dconfigure`
+    - On Windows: `ciphertool.bat -Dconfigure`
+
+3. Open the `deployment.toml` file again and see that the alias passwords are encrypted.
+
+    ```toml
+    [secrets]
+    admin_password = "GeNld2aZkydnIZGtkZYOnXlVzl8WBtZzAQ8kIoR5c7aHmyUkWTag7w4dG6B3JK5GxeX9bhsmZCBFozlPdWBT6Jvy"
```

Comment:
We can use the "xxxxx..." format here too

Reply:
fixed with https://github.com/hwupathum/docs-is/pull/5/commits/3df8725bb45a9cfc4c10b1ae1c1237de5715212f